sl/content.md

```diff
@@ -0,0 +1,89 @@
+# Scientific Linux
+
+%%LOGO%%
+
+This image containers for [Scientific Linux](http://www.scientificlinux.org/).
```

Comment:
This sentence seems a little wonky.  Maybe "These images contain Scientific Linux"?